### ISSUE TITLE
Images are saturated for adobe5k

### ISSUE THREAD

**Issue by deepakkupanda** [Li-Chongyi/Zero-DCE_extension#13]
Hi,

I trained ZDCE++ in adobe5k for 100 epochs. I had kept 4000 images for training, 500 for validation and 500 for testing. I used the original folder for training. I found the results to be **saturated**. 
One more question is on why we are not checking for validation loss. If we monitor validation loss, can this be improved.
Any help will be appreciated.

Thank you

![a0014-WP_CRW_6320](https://user-images.githubusercontent.com/41481181/161371981-d157b9d5-7ad4-433a-ac04-86e3c2ccd614.png)

![a0001-jmac_DSC1459](https://user-images.githubusercontent.com/41481181/161371983-a286adce-841e-4130-aaa2-bbe96d694d28.png)

![a0002-dgw_005](https://user-images.githubusercontent.com/41481181/161371978-ee7a8dc6-573e-4264-85bb-a91ae99e5d1a.png)

**Comment by deepakkupanda** [comment]
Hi 
Let me know if I am doing something wrong. Any help will be appreciated.
Thank you

**Comment by MoeLightOvO** [comment]
Maybe you should take some over-exposed images to trian your model. By the way, have you encountered noise problems? The model I trained has very obvious noise. I don't know why this happened.
